src/main/java/org/apache/accumulo/testing/performance/tests/DurabilityWriteSpeedPT.java

```diff
@@ -0,0 +1,110 @@
+package org.apache.accumulo.testing.performance.tests;
+
+import java.util.Arrays;
+
+import org.apache.accumulo.core.client.AccumuloClient;
+import org.apache.accumulo.core.client.BatchWriter;
+import org.apache.accumulo.core.client.admin.TableOperations;
+import org.apache.accumulo.core.conf.Property;
+import org.apache.accumulo.core.data.Mutation;
+import org.apache.accumulo.testing.performance.Environment;
+import org.apache.accumulo.testing.performance.PerformanceTest;
+import org.apache.accumulo.testing.performance.Report;
+import org.apache.accumulo.testing.performance.SystemConfiguration;
+
+public class DurabilityWriteSpeedPT implements PerformanceTest {
+  static final long N = 100000;
+
+  @Override
+  public SystemConfiguration getSystemConfig() {
+    return new SystemConfiguration();
+  }
+
+  private String[] init(AccumuloClient c) throws Exception {
+    String[] tableNames = getUniqueNames(4);
+    TableOperations tableOps = c.tableOperations();
+    createTable(c, tableNames[0]);
+    createTable(c, tableNames[1]);
+    createTable(c, tableNames[2]);
+    createTable(c, tableNames[3]);
+    // default is sync
+    tableOps.setProperty(tableNames[1], Property.TABLE_DURABILITY.getKey(), "flush");
+    tableOps.setProperty(tableNames[2], Property.TABLE_DURABILITY.getKey(), "log");
+    tableOps.setProperty(tableNames[3], Property.TABLE_DURABILITY.getKey(), "none");
+    return tableNames;
+  }
+
+  private void createTable(AccumuloClient c, String tableName) throws Exception {
+    c.tableOperations().create(tableName);
+  }
+
```

Comment:
It would be nice to set the table props at table creation time.  The following is one possible way to do this.  Also, the getUniqueNames function could be dropped.  That functions makes sense for ITs, but not here.

```java
  private String[] init(AccumuloClient c) throws Exception {
    String[] tableNames = new String[]{"syncT","flushT","logT","noneT"};
    
    createTable(c, tableNames[0], "sync");
    createTable(c, tableNames[1], "flush");
    createTable(c, tableNames[2], "log");
    createTable(c, tableNames[3], "none");
    
    return tableNames;
  }

  private void createTable(AccumuloClient c, String table, String durability) throws Exception {
    NewTableConfiguration ntc  = new NewTableConfiguration();
    ntc.setProperties(Collections.singletonMap("table.durability", durability));
    c.tableOperations().create(table, ntc);
  }
```